src/pages/model.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import App from '../../scripts/flux/components/app'
+
+export default React.createClass({
+  displayName: 'ModelPage',
+  render () {
+    return (
+      <App/>
+    )
```

Comment:
Expected an identifier and instead saw ')'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.